### DISCUSSION TITLE
Prevent users from downloading the file

### DISCUSSION THREAD

**Discussion by ndtai2510** [open-webui/open-webui#6136] [Q&A] [answered]
Currently, after the model answers from the document, when users view the citation of the answer, they can download the cited file. Is there a way to prevent users from downloading the file?
Thank you.

**Comment by silentoplayz** [comment, marked as answer]
I'm not able to reproduce on my end from a User account in Open WebUI. When the source within the citation for a document is clicked on from a User account, I am taken to a page like this:
![image](https://github.com/user-attachments/assets/a1a2b152-0667-43d5-8856-54900644731b)

**Reply by ndtai2510** [reply]
Great, I switched to the account with the role of "user," and as you mentioned, the attachment couldn't be uploaded. Thank you.
